python/kserve/kserve/model_server.py

```diff
@@ -132,30 +112,16 @@ def start(self, models: Union[List[Model], Dict[str, Deployment]], nest_asyncio:
         else:
             raise RuntimeError("Unknown model collection types")
 
-        if self.max_asyncio_workers is None:
-            # formula as suggest in https://bugs.python.org/issue35279
-            self.max_asyncio_workers = min(32, utils.cpu_count()+4)
-
-        self._http_server = tornado.httpserver.HTTPServer(
-            self.create_application(), max_buffer_size=self.max_buffer_size)
-
-        logging.info("Listening on port %s", self.http_port)
-        self._http_server.bind(self.http_port)
-        logging.info("Will fork %d workers", self.workers)
-        self._http_server.start(self.workers)
-
-        logging.info(f"Setting max asyncio worker threads as {self.max_asyncio_workers}")
-        asyncio.get_event_loop().set_default_executor(
-            concurrent.futures.ThreadPoolExecutor(max_workers=self.max_asyncio_workers))
```

Comment:
Do we still need this asyncio worker code for fastAPI ? I think it is not tornado specific.